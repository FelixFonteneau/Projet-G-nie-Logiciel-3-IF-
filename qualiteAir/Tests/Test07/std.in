1
45
45
1000
12/12/2017 12:12
2
80
90
10000
01/01/2017 00:00
12/12/2018 12:12
3
1
17/09/2017 10:00
19/11/2017 10:00
3
2
80
120
10000
29/09/2016 10:00
29/09/2019 23:00
4
100
80
55
5
6
./descriptionCapteursTest7_6.csv
./donneesCapteursTest7_6.csv
../../donnees/AttributeType.csv
N
7
