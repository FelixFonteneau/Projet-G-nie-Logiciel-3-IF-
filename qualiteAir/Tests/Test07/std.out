Bonjour, et bienvenue dans cette application de surveillance de la qualité de l'air.
Le service est en cours d'initialisation, veuillez patientez...

Nombre de mesures analysées : 20

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Quel est le rayon (en km) ? Format : 'nombres' et '.' seulement acceptés et radius > 0.


Quelle est la date (Au format JJ/MM/AAAA HH:MM) ?


Les quantités respectives de gaz/particules fines en moyenne dans la zone sont
(exprimées en microgrammes par mètre cube µg/m3):
Aucune mesure de O3 trouvée.
Aucune mesure de NO2 trouvée.
Aucune mesure de SO2 trouvée.
Aucune mesure de PM10 trouvée.


Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Quel est le rayon (en km) ? Format : 'nombres' et '.' seulement acceptés et radius > 0.


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


Les quantités respectives de gaz/particules fines en moyenne dans la zone sont
(exprimées en microgrammes par mètre cube µg/m3):
O3: 9.4168,
NO2: 37.0499,
SO2: 27.3914,
PM10: 5.49077,
Indice ATMO : 2 (Très bon)


Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


1 - Effectuer la requête sur tous les capteurs
2 - Effectuer la requête sur les capteurs d'un territoire délimité


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


           | Capteur 0 | Capteur 1 | Capteur 2 | Capteur 3 | Capteur 4 | 
------------------------------------------------------------------------
Capteur 0      100%         -1%         -1%         -1%         -1%   
------------------------------------------------------------------------
Capteur 1                  100%         -1%         -1%         -1%   
------------------------------------------------------------------------
Capteur 2                              100%         -1%         -1%   
------------------------------------------------------------------------
Capteur 3                                          100%         -1%   
------------------------------------------------------------------------
Capteur 4                                                      100%   
------------------------------------------------------------------------

(Un score de -1% signifie qu'aucune donnée n'a été trouvée sur cette periode là)

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


1 - Effectuer la requête sur tous les capteurs
2 - Effectuer la requête sur les capteurs d'un territoire délimité


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Quel est le rayon (en km) ? Format : 'nombres' et '.' seulement acceptés et radius > 0.


Quelle est la date de début (Au format JJ/MM/AAAA HH:MM) ?


Quelle est la date de fin (Au format JJ/MM/AAAA HH:MM) ?


           | Capteur 0 | Capteur 1 | 
------------------------------------
Capteur 0      100%        100%   
------------------------------------
Capteur 1                  100%   
------------------------------------

(Un score de -1% signifie qu'aucune donnée n'a été trouvée sur cette periode là)

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Mauvais format,
Quelle est la latitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -90 et 90.


Quelle est la longitude centrale du lieu souhaité ? Format : 'nombre' et '.' seulement acceptés, doit etre compris entre -180 et 180.


Il n'y a pas de capteurs dans un rayon de 90 km autour de l'endroit selectionné.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Il n'y a pas de capteur défaillant.

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Veuillez entrer le chemin d'accès du fichier de description des capteurs (absolu ou relatif)


Veuillez entrer le chemin d'accès du fichier de mesures (absolu ou relatif)


Veuillez entrer le chemin d'accès du fichier de description des types (absolu ou relatif)


Veuillez maintenant préciser si le fichier est encodé en utf8 ou non (Y/N)


Chargement des données en cours..

Nombre de mesures analysées : 20

Vous souhaitez :
1 - Obtenir la qualité de l'air moyenne sur un territoire précis à une date précise
2 - Obtenir la qualité de l'air moyenne sur un territoire précis sur une période précise
3 - Trouver les capteurs ayant un comportement similaire
4 - Obtenir les valeurs caractérisant la qualité de l'air à un point précis
5 - Obtenir la liste des capteurs qui ne fonctionnent pas
6 - Charger un autre fichier que celui par défaut
7 - Quitter


Merci et à bientôt !

